integrations/serverless-cost-management-integration.adoc

```diff
@@ -6,19 +6,19 @@ include::_attributes/common-attributes.adoc[]
 
 toc::[]
 
-link:https://access.redhat.com/documentation/en-us/cost_management_service/2022/html/getting_started_with_cost_management/assembly-introduction-cost-management#about-cost-management_getting-started[Cost management] is an {ocp-product-title} service that enables you to better understand and track costs for clouds and containers. It is based on the open source link:https://project-koku.github.io/[Koku] project.
+link:https://docs.redhat.com/en/documentation/cost_management_service/1-latest/html/getting_started_with_cost_management/index[Cost management] is an {ocp-product-title} service that enables you to better understand and track costs for clouds and containers. It is based on the open source link:https://project-koku.github.io/[Koku] project.
 
 [id="prerequisites_serverless-cost-management-integration"]
 == Prerequisites
 
 * You have cluster administrator permissions.
 
-* You have set up cost management and added an link:https://access.redhat.com/documentation/en-us/cost_management_service/2022/html/adding_an_openshift_container_platform_source_to_cost_management/index[{ocp-product-title} source].
+* You have set up cost management and added an link:https://docs.redhat.com/en/documentation/cost_management_service/1-latest/html/integrating_openshift_container_platform_data_into_cost_management/index[{ocp-product-title} source].
 
 include::modules/serverless-cost-management-labels.adoc[leveloffset=+1]
```

Comment:
Nitpick:
This is obviously out of the scope of this PR and up to you, but I recommend rethinking the heading of this reference module
from
`Using labels for cost management queries`
to
`Labels for cost management queries`
so that readers don't look for a procedure here (I couldn't find any procedure on this page after reading the *Prerequisites* section).

Reply:
I agree. The gerund form of the verb is typically preserved for PROCEDURE-type modules only.